change/@fluentui-react-charting-07085de4-08a0-49db-91db-76677708e0e3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31425&commitId=32da80031f4a40a067d42ab85379a7736656ec51&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-TreeChart</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-TreeChart.Basic.default.chromium.png|1223| Changed|
|react-charting-TreeChart.Basic - RTL.default.chromium.png|1223| Changed|
|react-charting-TreeChart.Compact.default.chromium.png|1223| Changed|
|react-charting-TreeChart.Compact - RTL.default.chromium.png|1223| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
VR screenshots expected to update due to change in color of tree edges.